sdk/identity/identity/src/msal/browserFlows/msalBrowserCommon.ts

```diff
@@ -195,4 +198,25 @@ export abstract class MsalBrowser extends MsalBaseUtilities implements MsalBrows
       return this.doGetToken(scopes);
     });
   }
+
+  /**
+   * Handles the MSAL authentication result.
+   * If the result has an account, we update the local account reference.
+   * If the token received is invalid, an error will be thrown depending on what's missing.
+   */
+  protected handleResult(
+    scopes: string | string[],
+    result?: MsalResult,
+    getTokenOptions?: GetTokenOptions,
+  ): AccessToken {
+    if (result?.account) {
```

Comment:
We're still understanding when and how this.account is used, so made this much more conservative by moving handleResult into the base classes and continuing to set this.account here. A larger refactor will remove a lot of this anyway, hopefully